release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -907,6 +913,84 @@ module openconfig-if-ip {
         "When set to true, router advertisement neighbor discovery
         messages are not transmitted on this interface.";
     }
+
+    leaf managed {
+      type boolean;
+      default false;
+      description
+        "When set to true, the managed address configuration (M) flag is set in
+        the advertised router advertisement. The M flag indicates that there are
+        addresses available via DHCPv6.";
+      reference "RFC4861: Neighbor Discovery for IPv6, section 4.2";
+    }
+
+    leaf other-config {
+      type boolean;
+      default false;
+      description
+        "When set to true, the other configuration (O) flag is set in the
+        advertised router advertisement. The O flag indicates that there is
+        other configuration available via DHCPv6 (e.g., DNS servers).";
+      reference "RFC4861: Neighbor Discovery for IPv6, section 4.2";
+    }
+  }
+
+  grouping ipv6-ra-prefix-config {
```

Comment:
It might be worth considering to have some of these leaves be defaulted to what is defined in https://datatracker.ietf.org/doc/html/rfc4861#section-6.2.1

Reply:
Some of the leafs above for the configuratoin of ipv6 router advertisements per interface do have defaults as specified in the rfc.  But the options for prefixes ([rfc4861 section-4.6.2](https://datatracker.ietf.org/doc/html/rfc4861#section-4.6.2)) don't seem to have defaults?